### PR TITLE
Parsing address in asan report

### PR DESCRIPTION
**Pull request by Avgor46**

### CODE REVIEW

File: src/execution_class.rs [hunk 1]
```diff
@@ -131,12 +131,18 @@ impl<'a> ExecutionClass<'a> {
     /// * `short_desc` - short description of execution class.
     ///
     /// * `rw` - access information.
-    pub fn san_find(short_desc: &'a str, rw: Option<&'a str>) -> error::Result<Self> {
+    pub fn san_find(
+        short_desc: &'a str,
+        rw: Option<&'a str>,
+        near_null: bool,
```
Comment (SweetVishnya) [inline src/execution_class.rs:137]: parameter docs

File: src/bin/casr-san.rs [hunk 2]
```diff
@@ -173,14 +174,33 @@ fn main() -> Result<()> {
...
+                            } else {
+                                None
+                            }
                         } else {
                             None
                         };
 
-                        if let Ok(class) = ExecutionClass::san_find(san_type, mem_access) {
+                        let rcrash_address =
+                            Regex::new("on (?:unknown )?address 0x([0-9a-f]+)").unwrap();
```
Comment (SweetVishnya) [inline src/bin/casr-san.rs:191]: maybe `on.*address....` ?

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: src/execution_class.rs (hunk 1 above)
Comment (Avgor46) [reply src/execution_class.rs:139]: Done

File: src/bin/casr-san.rs (hunk 2 above)
Comment (Avgor46) [reply src/bin/casr-san.rs]: Done

**Review by SweetVishnya — APPROVED** [review]

**Pull request merged**